tests/test_forms.py

```diff
@@ -0,0 +1,18 @@
+from django.test import TestCase
+
+from taxi.forms import DriverCreationForm
+
+
+class FormsTest(TestCase):
+    def test_driver_creation_form_with_license_number_is_valid(self):
+        form_data = {
+            "username": "new_user",
+            "password1": "pass12345",
+            "password2": "pass12345",
+            "first_name": "Test First name",
+            "last_name": "Test Last name",
+            "license_number": "ABC12345",
+        }
+        form = DriverCreationForm(data=form_data)
+        self.assertTrue(form.is_valid())
+        self.assertEqual(form.cleaned_data, form_data)
```

Comment:
I think you can add tests if you insert invalid license number. And also tests for raising ValidationErrors with different message depending on problems with license_number